frontend/src/components/jobs/table/utils.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { UncontrolledPopover, Card, CardHeader, CardBody } from "reactstrap";
+import { MdInfo } from "react-icons/md";
+import PropTypes from "prop-types";
+
+export function PlaybookExecutedInfoCard({ job }) {
+  return (
+    <Card className="border-dark p-0">
+      <CardHeader className="d-flex align-items-center bg-body p-2 h6">
+        <span>{job.playbook_to_execute || "Custom analysis"}</span>
+      </CardHeader>
+      <CardBody className="bg-darker border-top border-tertiary">
+        <div className="d-flex flex-column justify-content-center">
+          <span>
+            {job.analyzers_to_execute.length}/{job.analyzers_requested.length}{" "}
+            analyzers
+          </span>
+          <span>
+            {job.connectors_to_execute.length}/{job.connectors_requested.length}{" "}
+            connectors
+          </span>
+          <span>{job.pivots_to_execute.length}/all pivots</span>
+          <span>{job.visualizers_to_execute.length}/all visualizers</span>
```

Comment:
I suggest to make pivot and visualizer denominators like analyzer and connector.